For utils/pdftocairo, move common_libs to the front to avoid picking
up any already installed older version of libpoppler.so.

Index: utils/CMakeLists.txt
--- utils/CMakeLists.txt.orig
+++ utils/CMakeLists.txt
@@ -36,7 +36,7 @@ if (HAVE_CAIRO)
   )
   add_definitions(${CAIRO_CFLAGS})
   add_executable(pdftocairo ${pdftocairo_SOURCES})
-  target_link_libraries(pdftocairo ${CAIRO_LIBRARIES} Freetype::Freetype ${common_libs})
+  target_link_libraries(pdftocairo ${common_libs} ${CAIRO_LIBRARIES} Freetype::Freetype)
   target_include_directories(pdftocairo SYSTEM PRIVATE ${CAIRO_INCLUDE_DIRS})
   if(LCMS2_FOUND)
     target_link_libraries(pdftocairo ${LCMS2_LIBRARIES})
